frontend/src/components/custom-tools/prompt-card/PromptOutput.jsx

```diff
@@ -351,16 +341,25 @@ function PromptOutput({
                           <PlayCircleFilled className="prompt-card-actions-head" />
                         </Button>
                       </Tooltip>
-                      <Tooltip title="Expand">
-                        <Button
-                          size="small"
-                          type="text"
-                          className="prompt-card-action-button"
-                          onClick={() => handleExpandClick(profile)}
-                        >
-                          <ArrowsAltOutlined className="prompt-card-actions-head" />
-                        </Button>
-                      </Tooltip>
+                      <CopyPromptOutputBtn
+                        isDisabled={enforceType === TABLE_ENFORCE_TYPE}
+                        copyToClipboard={() =>
+                          copyOutputToClipboard(
+                            displayPromptResult(
+                              result.find(
```

Comment:
NIT:
@tahierhussain Lack context, If the `result` array will be large, its better to use `map` and `get`
Usage of search/find in similar manner could take a toll on performance when `result` is large.

Reply:
@athul-rs The `result` array stores the output of the prompt for each LLM profile. Since we allow only three LLM profiles per project, the length of the `result` array will never exceed 3.

Additionally, the `find` function stops executing once it finds the first matching element, making it potentially faster if the desired element is near the beginning of the array. In contrast, the `map` function processes every element in the array, making it less optimal in comparison. I hope this clarifies your doubt.